ebos/ecloutputblackoilmodule.hh

```diff
@@ -114,14 +119,29 @@ class EclOutputBlackOilModule
             GasInPlaceInGasPhase = 6,  //GIPG
             PoreVolume = 7, //PV
         };
-        static const int numFipValues = PoreVolume + 1 ;
+        static const int numFipTypes = PoreVolume + 1 ;
+
+
+        static std::string EclString(int fip_type) {
+            switch(static_cast<FipId>(fip_type)) {
+            case FipDataType::WaterInPlace: return "WIP";
+            case FipDataType::OilInPlace: return "OIP";
+            case FipDataType::GasInPlace: return "GIP";
+            case FipDataType::OilInPlaceInLiquidPhase: return "OIPL";
+            case FipDataType::OilInPlaceInGasPhase: return "OIPG";
+            case FipDataType::GasInPlaceInLiquidPhase: return "GIPL";
+            case FipDataType::GasInPlaceInGasPhase: return "GIPG";
+            case FipDataType::PoreVolume: return "PV";
+            }
+            throw std::logic_error("fip_type:  " + std::to_string(fip_type) + " not recognized");
```

Comment:
Please include `<stdexcept>` for `std::logic_error`.

Reply:
OK